### ISSUE TITLE
question on train_classifier

### ISSUE THREAD

**Issue by UestcJay** [mrakotosaon/dse-meshing#3]
Thanks for your great work! 
I don’t understand part of the code in `train_classifier.py` very well. 
1.  In dataset, `gt_map = data[:,:N_NEIGHBORS,3:]`   how can we make the gt_map based on the point cloud?
2.  In the paper,  L2 loss is $|| c_j - sigmoid(\hat{c}_j) ||$，where $\hat{c}_j$is the ground classification and c_j is  classification network outputs,  Corresponding to the code `class_loss = tf.reduce_mean(tf.square(labels - tf.nn.sigmoid(map)))*30.0`？  but `map` is the network output, `labels` Is based on `gt_map`,  is it the reverse or I read it wrong？
Looking forward to your reply!

**Comment by mrakotosaon** [comment]
Hi @UestcJay 
1. The dataset is a collection of 3D patches around center points. The vertices are sampled uniformly from a given mesh. The first 3 columns of the last dimension correspond to 3D coordinates. The last 2 columns correspond to logarithmic maps computed from the method of Nicholas Sharp et al. "The vector heat method."  https://github.com/nmwsharp/vector-heat-demo. We first remesh the ground truth meshes to contain sufficient vertices. We then apply the vector heat method centered at centers of patches to compute logmaps and  save the results for the nearest Euclidean 200 vertices inside point clouds sampled at 10K vertices for each ground truth mesh. 
2. Thank you for pointing that out. There is an error in the paper, the sigmoid should be reversed, like in the code.
